package.json

```diff
@@ -1,56 +1,66 @@
-    "docs": "typedoc src --out docs",
-    "test": "jest --coverage",
-    "prepare": "husky install"
-  },
-  "lint-staged": {
-    "**/*": "prettier --write --ignore-unknown"
-  },
-  "jest": {
-    "preset": "ts-jest"
-  },
-  "keywords": [
-    "bb2",
-    "cms",
-    "medicare",
-    "claims"
-  ],
-  "author": "",
-  "license": "ISC",
-  "devDependencies": {
-    "@rollup/plugin-commonjs": "^21.0.1",
-    "@rollup/plugin-node-resolve": "^13.1.3",
-    "@rollup/plugin-typescript": "^8.3.0",
-    "@types/jest": "^27.4.0",
-    "@typescript-eslint/eslint-plugin": "^5.10.0",
-    "@typescript-eslint/parser": "^5.10.0",
-    "eslint": "^8.7.0",
-    "eslint-config-prettier": "^8.3.0",
-    "husky": "^7.0.4",
-    "jest": "^27.4.7",
-    "lint-staged": "^12.2.2",
-    "prettier": "2.5.1",
-    "rollup": "^2.64.0",
-    "ts-jest": "^27.1.3",
-    "ts-node": "^10.4.0",
-    "tslib": "^2.3.1",
-    "typescript": "^4.5.4"
-  },
-  "dependencies": {
-    "@microsoft/api-extractor": "^7.19.4",
-    "typedoc": "^0.22.11"
-  }
-}
+    "name": "cms-bb2",
+    "version": "1.0.0",
+    "description": "An sdk used for interacting with the CMS Blue Button 2.0 API",
+    "main": "dist/my-lib.cjs.js",
+    "module": "dist/my-lib.esm.js",
+    "browser": "dist/my-lib.umd.js",
+    "files": [
+        "dist"
+    ],
+    "scripts": {
+        "dev": "rollup -c -w",
+        "build": "rollup -c",
+        "build:types": "tsc -p ./tsconfig.json --outDir build --declaration true && api-extractor run",
+        "docs": "typedoc src --out docs",
+        "test": "jest --coverage",
+        "prepare": "husky install"
+    },
+    "lint-staged": {
+        "**/*": "prettier --write --ignore-unknown"
+    },
+    "jest": {
+        "preset": "ts-jest"
+    },
+    "keywords": [
+        "bb2",
+        "cms",
+        "medicare",
+        "claims"
+    ],
+    "author": "",
+    "license": "ISC",
+    "devDependencies": {
+        "@rollup/plugin-commonjs": "^21.0.1",
+        "@rollup/plugin-node-resolve": "^13.1.3",
+        "@rollup/plugin-typescript": "^8.3.0",
+        "@types/jest": "^27.4.0",
+        "@typescript-eslint/eslint-plugin": "^5.10.0",
+        "@typescript-eslint/parser": "^5.10.0",
+        "axios": "^0.26.0",
+        "cookie-parser": "^1.4.6",
+        "eslint": "^8.7.0",
+        "eslint-config-prettier": "^8.3.0",
+        "express-async-errors": "^3.1.1",
+        "http-status-codes": "^2.2.0",
+        "husky": "^7.0.4",
+        "jest": "^27.4.7",
+        "lint-staged": "^12.2.2",
+        "prettier": "2.5.1",
+        "rollup": "^2.64.0",
+        "ts-jest": "^27.1.3",
+        "ts-node": "^10.4.0",
+        "tsconfig-paths": "^3.12.0",
+        "tslib": "^2.3.1",
+        "typescript": "^4.5.4"
+    },
+    "dependencies": {
+        "@microsoft/api-extractor": "^7.19.4",
+        "@types/cookie-parser": "^1.4.2",
+        "@types/express": "^4.17.13",
```

Comment:
We shouldn't be importing `express` or the express types. The reason for this is that it is a web server. The SDK shouldn't require anything from a web server at this point. We need to keep the SDK agnostic. For example another web server that is used: https://github.com/koajs/koa . If we only support express it's putting a restriction on the 3rd party app that is integrating with us that is unnecessary.